specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2019-04-01-preview/managedservices.json

```diff
@@ -0,0 +1,902 @@
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Put Registration Assignment": {
+            "$ref": "./examples/PutRegistrationAssignment.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.ManagedServices/registrationDefinitions": {
+      "get": {
+        "tags": [
+          "RegistrationDefinitions"
+        ],
+        "operationId": "RegistrationDefinitions_List",
+        "description": "Gets a list of the registration definitions.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns a list of the registration definitions.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationDefinitionList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get Registration Definitions": {
+            "$ref": "./examples/GetRegistrationDefinitions.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.ManagedServices/registrationAssignments": {
+      "get": {
+        "tags": [
+          "RegistrationAssignments"
+        ],
+        "operationId": "RegistrationAssignments_List",
+        "description": "Gets a list of the registration assignments.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/ExpandRegistrationDefinition"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns a list of the registration assignments.",
+            "schema": {
+              "$ref": "#/definitions/RegistrationAssignmentList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get Registration Assignments": {
+            "$ref": "./examples/GetRegistrationAssignments.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.ManagedServices/operationstatuses/{operationId}": {
```

Comment:
this does not need to be modeled in swagger. The value comes from the header.

Reply:
Removed